### ISSUE TITLE
The Loss function

### ISSUE THREAD

**Issue by Oussamab21** [AaronX121/Soft-Decision-Tree#2]
I was just wondering that you used standard cross entropy loss instead of the loss used in Hinton's paper because that loss is used only when we have a neural network as teacher model right?
and why you are not using the penalty term calculated ? at the line 30 in the file SDT.py ?

**Comment by xuyxu** [comment]
Thanks for pointing out this mistake, I have fixed it by adding the penalty term onto the loss in "main.py".

**Comment by Oussamab21** [comment]
Thank you but the other question was why didn't you use the same loss function used in the paper and used the classical cross entropy instead ?

**Comment by xuyxu** [comment]
As mentioned in issue #1, my target is NOT to conduct Knowledge Distillation (KD). Soft decision tree also has some other uses, such as the research done in best paper of ICCV-2015 "Deep Neural Decision Forest"
